index.js

```diff
@@ -24,7 +24,8 @@ if (process.env.NODE_ENV === 'production') {
 }
 
 // Configure the routes
-app.post(`${config.CONTEXT_PATH}/scan`, ClamAVControler.scan)
+app.post(`${config.CONTEXT_PATH}/scan`, ClamAVController.scan)
+app.get(`${config.CONTEXT_PATH}/health`, ClamAVController.check)

```

Comment:
@BadalZ @cwdcwd Note that the health endpoint here is `/api/v1/health`. The prefix is a configuration. Let me know if it needs to only be `/health` instead.

Reply:
I think ECS _might_ be able to do that path. We might need to remove the `/api/v1` anyways if we route this behind the Tyk gateway.

Reply:
@callmekatootie did you not want to use the [middleware](https://www.npmjs.com/package/topcoder-healthcheck-dropin) we built?

Reply:
`/api/v1/` may be removed in AWS since we place the services behind the Tyk API gateway